connectors/src/lib/data_sources.ts

```diff
@@ -599,6 +601,7 @@ export async function upsertTableFromCsv({
     tableId,
     truncate,
     async: true,
+    useAppForHeaderDetection,
```

Comment:
I don't understand this one as I don't see a diff on any v1 endpoint?

Reply:
The v1 api actually calls the handler from private api ( from `front/pages/api/w/[wId]/data_sources/[dsId]/tables/csv.ts` ) and is undocumented ("System API key only endpoint. Undocumented.")

Reply:
It certainly does not as per the endpoint URL above

Reply:
As per IRL the handler is shared